.gitignore

```diff
@@ -1,3 +1,3 @@
 export/
 .vscode/
-APIStuff.hx
```

Comment:
If you're gonna do this, you should instead use environment variables, not a class containing two strings.

Reply:
Do you have a block of code I can add to it